libcxx/utils/libcxx/test/format.py

```diff
@@ -424,3 +424,18 @@ def _splitFile(self, input):
             thisFileContent.append(line)
         if currentFile is not None:
             yield (currentFile, '\n'.join(thisFileContent))
+
+# Workaround for https://gitlab.kitware.com/cmake/cmake/-/issues/19180.
```

Comment:
https://gitlab.kitware.com/cmake/cmake/-/issues/19180 is only applicable if you use the default `CMAKE_{C,CXX}_COMPILER`, that is `/usr/bin/{cc,c++}`. If you change the compiler, for example when using the bootstrapping build, you'll still need to either use `xcrun` (or manually pass `-isysroot`) so even if https://gitlab.kitware.com/cmake/cmake/-/issues/19180 eventually gets resolved, you won't be able to remove this.